sbndqm/dqmAnalysis/TPC/SBND/OnlineEvdSBND_module.cc

```diff
@@ -0,0 +1,431 @@
+///////////////////////////////////////////////////////////////////////////
+// Class:       OnlineEvdSBND
+// Plugin Type: analyzer
+// File:        OnlineEvdSBND_module.cc
+// Author:      tjyang@fnal.gov
+// Analyzer module to make event display for DQM
+//
+// April 22, 2024: Add two png files to show RMS in the WIB and FEM space
+///////////////////////////////////////////////////////////////////////////
+#include <vector>
+#include <chrono>
+
+#include "TROOT.h"
+#include "TTree.h"
+#include "TH2D.h"
+#include "TStyle.h"
+#include "TCanvas.h"
+#include "TLatex.h"
+#include "TTimeStamp.h"
+#include "TLine.h"
+
+#include "art/Framework/Core/EDAnalyzer.h"
+#include "art/Framework/Core/ModuleMacros.h"
+#include "larcore/Geometry/Geometry.h"
+
+#include "canvas/Utilities/InputTag.h"
+#include "art/Framework/Principal/Event.h"
+#include "art/Framework/Principal/Handle.h"
+#include "art/Framework/Principal/Run.h"
+#include "art/Framework/Principal/SubRun.h"
+#include "art_root_io/TFileService.h"
+
+#include "sbndqm/dqmAnalysis/TPC/ChannelDataSBND.hh"
+#include "sbndqm/Decode/TPC/HeaderData.hh"
+#include "sbndqm/dqmAnalysis/TPC/AnalysisSBND.hh"
+
+//#include "sbndqm/Decode/TPC/SBND/ChannelMap/TPCChannelMapService.h"
+#include "sbndcode/ChannelMaps/TPC/TPCChannelMapService.h"
```

Comment:
I'm getting a compilation error from this include statement... we don't have `sbndcode` available at ICARUS.
I suppose the lines got mixed up? You want to include the local channel mapping service, right?